src/components/CompleteProfile/CompleteProfile.css

```
@@ -0,0 +1,401 @@
+				width: 14px;
+			}
+
+			.slider-checkbox:checked + label::after {
+				background-color: white;
+				left: 20px;
+			}
+		}
+	}
+
+	.details-content-file-upload {
+		align-items: center;
+		display: flex;
+		gap: var(--spacing-sm);
+		justify-content: center;
+
+		& picture {
+			max-height: 128px;
+			max-width: 128px;
+			min-height: 52px;
+			min-width: 52px;
+
+			& img {
+				aspect-ratio: 1;
+				border-radius: 0.5rem;
+				height: 100%;
+				object-fit: cover;
+				width: 100%;
+			}
+		}
+
+		& .details-file-upload-buttons-wrapper {
+			display: flex;
+			flex-direction: column;
+			gap: var(--spacing-xs);
+			justify-content: start;
+			min-width: 244px;
+			width: 244px;
+
+			& input[type='file'] { display: none; }
+
+			& .file-upload-success {
+				background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='10' height='10' viewBox='0 0 24 24' fill='none'%3E%3Cpath d='M20 6L9 17L4 12' stroke='%23009900' stroke-width='2' stroke-linecap='round' stroke-linejoin='round'/%3E%3C/svg%3E");
+				background-position: start;
+				background-repeat: no-repeat;
+				background-size: contain;
+				color: var(--color-success);
+				content: '';
+				font-size: var(--text-xs);
+				font-weight: var(--weight-regular);
+				letter-spacing: 0;
+				line-height: 160%;
+				padding-left: 1.5rem;
+			}
+
+			& .custom-file-upload {
+				border-radius: var(--rounded-md);
+				color: var(--color-black);
+				cursor: pointer;
+				font-size: var(--text-xs);
+				font-weight: var(--weight-semibold);
+				letter-spacing: 0;
+				line-height: 160%;
+				padding: var(--spacing-xs) var(--spacing-sm);
+				text-align: center;
+			}
+
+			label.custom-file-upload { background-color: var(--color-card); }
+
+			button.custom-file-upload {
+				background-color: var(--color-background);
+				border: var(--border);
+				border-color: var(--color-black);
+			}
+		}
+	}
+
+	#details-information-grid {
+		display: grid;
+		gap: var(--spacing-sm);
+		grid-template-columns: repeat(3, 1fr);
+		justify-content: space-between;
+
+		label[for='linkedInProfile'], label[for='githubProfile'], label[for='portfolio'] {
+			display: inline-block;
+			margin-bottom: var(--spacing-xs);
+		}
+
+		#details-information-skills {
+			display: flex;
+			flex-direction: column;
+			grid-column: 1 / span 3;
+
+			> label {
+				font-size: var(--text-base);
+				letter-spacing: 0;
+				line-height: 160%;
+			}
+
+			#skills {
+				align-items: baseline;
+				background-color: var(--color-card);
+				border: var(--border-base);
+				border-color: var(--color-border);
+				border-radius: var(--rounded-base);
+				border-style: solid;
+				display: flex;
+				flex-wrap: wrap;
+				gap: var(--spacing-xs);
+				min-height: 4rem;
+				padding: var(--spacing-xs);
+				resize: none;
+
+				> span {
+					align-items: center;
+					background-color: var(--color-accent);
+					border: solid var(--border-base) var(--color-accent);
+					border-radius: 99px;
```

```suggestion
					border-radius: 100vmax;
```

Neat little trick to make things round 😉

It uses the largest viewport unit (`vmax`) and makes it as big as the whole viewport (100 of the unit).